index.js

```diff
@@ -475,6 +479,9 @@ function guessSupportedScheme(keySystem) {
  * @return {boolean} True if browser natively supports encryptionScheme.
  */
 function hasEncryptionScheme(mediaKeySystemAccess) {
+  if (!mediaKeySystemAccess) {
```

Comment:
It seems like this would apply the workaround in cases where it shouldn't be, by giving a false negative for unsupported configs.  We should check for capabilities.supported or capabilities.keySystemAccess before calling this method.